singular.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * The template for displaying all pages, single posts and attachments
+ *
+ * This is a new template file that WordPress introduced in
+ * version 4.2. Note that it uses conditional logic to display
```

Comment:
Another good point. Twenty Sixteen already requires 4.2, so maybe I'll test on 4.2 to make sure index.php works fine as a fallback for single.php and page.php